typhon/display.py

```diff
@@ -59,19 +59,18 @@ class TyphonDisplay(TyphonBase, TyphonDesignerMixin, TemplateTypes):
     parent: QWidget, optional
     """
     # Template types and defaults
-    Q_ENUMS(TemplateTypes)
-    TemplateEnum = TemplateTypes.to_enum()  # For convenience
-    default_templates = dict((_typ.name, os.path.join(ui_dir,
-                                                      _typ.name + '.ui'))
-                             for _typ in TemplateEnum)
+    Q_ENUMS(DisplayTypes)
+    TemplateEnum = DisplayTypes.to_enum()  # For convenience
 
     def __init__(self,  parent=None, **kwargs):
         # Intialize background variable
-        self._use_template = ''
-        self._use_default = False
+        self._forced_template = ''
         self._last_macros = dict()
         self._main_widget = None
-        self._template_type = TemplateTypes.detailed_screen
+        self._display_type = DisplayTypes.detailed_screen
+        self.templates = dict((_typ.name, os.path.join(ui_dir,
```

Comment:
`ui_dir` -> `typhon_package_ui_path` or something?

Reply:
hmmmmm, that is an awfully vague name